### PR TITLE
Added complete Dutch translation

### PR DESCRIPTION
**Pull request by mourix**
I've translated all the strings from the locales template into Dutch. This should be ready for use. 

On a sidenote, could someone refresh the gettext template strings, in order for me to add the missing words?

Have a good day!

### CODE REVIEW

**Comment by lightningterror** [comment]
Can you update the copyright headers for the .mo files ?

> # Translations for PCSX2 (Iconized).
> # Copyright (C) 2018 PCSX2 Dev Team
> # This file is distributed under the same license as the PCSX2 package.

**Comment by mourix** [comment]
I guess you meant .po files? Done!

**Comment by mourix** [comment]
Done. I guess I should have looked beyond the poedit app.

File: locales/nl_NL/pcsx2_Iconized.po [hunk 1]
```diff
@@ -0,0 +1,871 @@
+# Translations for PCSX2 (Iconized).
+# Copyright (C) 2018 PCSX2 Dev Team
+# This file is distributed under the same license as the PCSX2 package.
+# First author: mourix, 2018
```
Comment (lightningterror) [inline locales/nl_NL/pcsx2_Iconized.po]: Add an extra line with # after this one to both files.
```

Should be 
# Translations for PCSX2 (Iconized).
# Copyright (C) 2018 PCSX2 Dev Team
# This file is distributed under the same license as the PCSX2 package.
# First author: mourix, 2018
#
```

Comment (mourix) [reply locales/nl_NL/pcsx2_Iconized.po]: Done.

**Comment by pgert** [comment]
@mourix
The dutch translation needs a big update.
Do you know how to make a GitHub repository in you harddrive, and how to update the PO-files from it?

**Comment by pgert** [comment]
@mourix
I assume you have Windows ...

1 Download TortoiseSVN from this address: "https://tortoisesvn.net/downloads.html" and install it.
2 Make a folder somewhere on your harddrive, and call it for example "GitHub - pcsx2".
3 Rightclick the folder, and choose "SVN Checkout...".
4 In the dialog that pops up, paste "https://github.com/PCSX2/pcsx2/trunk" into "URL of repository:", and click OK. TortoiseSVN will now download the files for pcsx2.
5 Open Iconized & Main (seperately) with a (deasent) texteditor, and replace "X-Poedit-Basepath: trunk\n" with (as an example) "X-Poedit-Basepath: C:/GitHub - pcsx2\n", and save.
6 Open Iconized & Main (seperately) with POedit, and click "Update from code".

**Comment by refractionpcsx2** [comment]
Don't you mean TortoiseGit?

**Comment by pgert** [comment]
@refractionpcsx2
No, I use TortoiseSVN.
TortoiseGit is new to me.

**Comment by mourix** [comment]
@pgert I already used Github Desktop and I figured it now. It is gonna take a bit to get it all done but I'll update in a while. Thanks for the help.

**Pull request merged**